modules/http/http.h

```diff
@@ -93,5 +96,7 @@ void http_dd_set_body_suffix(LogDriver *d, const gchar *body_suffix);
 void http_dd_set_delimiter(LogDriver *d, const gchar *delimiter);
 void http_dd_insert_response_handler(LogDriver *d, HttpResponseHandler *response_handler);
 LogTemplateOptions *http_dd_get_template_options(LogDriver *d);
+void http_dd_set_accept_encoding(LogDriver *d, const gchar *encoding);
+void http_dd_set_message_compression(LogDriver *d, const gchar *encoding);
```

Comment:
Copy-paste error in the input parameter (encoding -> compression)

Reply:
cURL terminology uses `accept_encoding` to refer to HTTP response compression. Since compressions are a subset  of encodings, I think the naming is correct here. If you insist on changing the argument name, I would also suggest changing the name of `http_dd_set_accept_encoding()` to make the naming more homogeneous.